CODINGSTYLE.md

```diff
@@ -181,6 +181,7 @@ switch (a) {
 	case 1:
 		DoSomething();
 		/* FALL THROUGH */
```

Comment:
Now this comment can also be removed, as the macro alone is more than clear enough :)

Reply:
Ok!

Reply:
I was about to comment the same and ask whether should I remove it :)